onboarding/src/Components/Employee/EmployeeFormPages/EmployeeSingleFormPage.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import EmployeeSections from "./EmployeeSections";
+import PropTypes from "prop-types";
+
+const EmployeeSinglePage = ({ page }) => {
+  const pageLink = page.link ? page.link.replace(/watch\?v=/g, "embed/") : "";
```

Comment:
Do we expect that _youTube_ will be always here(?)  ;-) :-)
Let's think about `if`-statement with regex like
`/^(?:(?:(?:https?:)?\/\/)?(?:www\.)?(?:youtu(?:be\.com|\.be))\/(?:watch\?v\=|v\/|embed\/)?([\w\-]+))/i`
(and maybe something for _vimeo_ also :-) )
If it is _youTube_, use current solution
otherwise e.g.
`<a href="pageLink">page.title</a>`
and we are good to go :-) :-)